src/modules/scores.js

```diff
@@ -0,0 +1,64 @@
+// Get HTML elements
+const addButton = document.querySelector('.add-button');
+const refreshButton = document.querySelector('.refresh');
+const errorElement = document.getElementById('error');
+
+// Add event listeners
+
+// Refresh button to reload the page and clear the form
+const refreshScores = () => {
+  document.querySelector('.add-form').reset();
+  window.location.reload();
+};
+
+// Add new score to the list and save them into localStorage
+const addScore = () => {
+  const nameInput = document.querySelector('#name').value;
+  const scoreInput = document.querySelector('#score').value;
+
+  if (nameInput && scoreInput && scoreInput !== 0 && scoreInput > 0) {
+    const scores = JSON.parse(localStorage.getItem('scores')) || [];
+    scores.push({ nameInput, scoreInput });
+
+    localStorage.setItem('scores', JSON.stringify(scores));
+    document.querySelector('.add-form').reset();
+    window.location.reload();
+  } else if (scoreInput === 0 || scoreInput < 0) {
+    errorElement.innerHTML = '<p>Score must be greater than 0</p>';
+    setTimeout(() => {
+      errorElement.innerHTML = '';
+    }, 3000);
+  } else {
+    errorElement.innerHTML = '<p>Please fill all the fields</p>';
+    setTimeout(() => {
+      errorElement.innerHTML = '';
+    }, 3000);
+  }
+};
+
+addButton.addEventListener('click', addScore);
+refreshButton.addEventListener('click', refreshScores);
```

Comment:
- [ ] Great job on implementing the setting up of your project as specified in the project requirement! 👏 However, you seem to have not understood the scope of this milestones project requirements properly and due to that, you went beyond the project requirements which is really impressive but for us to fulfill the requirements kindly make sure to do exactly as required. 
  - [ ] Please note that the main goal of this milestone is for you to learn how to set up your `webpack` for this project. No`Add`, `Remove`, or `API` functionality is to be implemented in this milestone. You only need to **write a plain HTML markup with minimum styling (just to make the layout).** See below for more details on project requirements: 
  
<img width="1082" alt="Screenshot 2023-06-22 at 00 22 49" src="https://github.com/Ruthmy/leaderboardProject/assets/107192076/5fa770c5-640d-47e7-8731-892a13b19956">

Reply:
Hi @Strangeal ! Thanks for your feedback.
I fully understood the scope of the project. As the description of my PR says, I added functionality for the fun of practicing, since it seems that was a problem, I removed it.